src/main/java/innohack/gem/entity/rule/rules/TikaMetaValue.java

```diff
@@ -0,0 +1,83 @@
+package innohack.gem.entity.rule.rules;
+
+import com.google.common.collect.Lists;
+import innohack.gem.entity.GEMFile;
+import innohack.gem.entity.feature.AbstractFeature;
+import innohack.gem.entity.feature.TikaFeature;
+import innohack.gem.entity.rule.ParamType;
+import innohack.gem.entity.rule.Parameter;
+import innohack.gem.entity.rule.RuleType;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+
+public class TikaMetaValue extends Rule {
+
+  static final String LABEL = "Tika Meta Regex";
+  static final RuleType RULE_TYPE = RuleType.TIKA_METADATA;
+  static final List<Parameter> PARAMETERS =
+      Lists.newArrayList(new Parameter("headers", "aa,bb,cc", ParamType.STRING_LIST));
+
+  public TikaMetaValue() {
+    this.setLabel(LABEL);
+    this.setRuleType(RULE_TYPE);
+    this.setParams(PARAMETERS);
```

Comment:
the default constructor should take 2 string parameter. call `this(null, null)` instead